### ISSUE TITLE
NVDA is announcing 'Document' instead of announcing visual status message 'AI is analyzing  :A11y_DocumentDB for VS Code Extension_View Query Insights_Screenreader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode-documentdb#380]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #FTP; #Win32; #DocumentDB for VS Code Extension; #SH_DocumentDB_Win32_NOV2025; #WCAG4.1.3; #Screenreader; #NVDA; #Closed; #A11yFixed;

## Environment Details:
Application Name: DocumentDB for VS Code Extension
Version 0.6.0
OS: Windows 11[Version 23H2(OS Build 22631.3593)]  
NVDA 2025

## Repro Steps:

1. Open VS code 
2. Install the DocumentDB for VS Code Extension and open it
3. Navigate to the DocumentDB Connections area.
4. Click the New Connection action.
5. Select Connection String as the connection method.
6. Paste the connection string when prompted.
7. Choose to authenticate with username/password and enter the credentials
8. Confirm to create the connection.
9. Locate the desired cluster in the DocumentDB Connections list.
10. Expand the cluster entry to initiate the connection.
11. Expand one of the databases under the cluster.
12. Expand a collection within that database.
13. Select the Documents item for that collection.
14. Collection View will open
15. Run the query: { "cuisine": "Thai" } 
16. TAB till "query insights" tab and press ENTER key
17. TAB till "Get AI performance insights" button and press ENTER key
18. Observe the issues

# Actual Result: 
When we hitting ENTER on "Get AI performance insights" button visually "AI is analyzing..."  but NVDA instead of announcing visual status message announcing 'document'.
 
## Expected Result:
When we hitting ENTER on "Get AI performance insights" button, Instead of 'document' NVDA should announce the visual status message

## User Impact:
Screen reader user will not the status after hitting ENTER on  "Get AI performance insights" button. 

## Attachment

https://github.com/user-attachments/assets/52258e66-3eed-45be-b77a-7efcf33fe2eb
